cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
We do this a lot in the docker release scripts but something like 

if [ "$(cat NOTARY_VERSION)" = *-dev]; then
     # add -X notaryservervar is dev server
else
    # make it prod
     # for a real release version
fi

something like that

Btw this is from my phone so that probably looks like poo


> On Aug 7, 2015, at 22:28, Jessie Frazelle <jessica.frazelle@docker.com> wrote:
> 
> Maybe it should be a compile time variable :/
> 
>> On Aug 7, 2015, at 22:20, Nathan McCauley <notifications@github.com> wrote:
>> 
>> In cmd/notary/main.go:
>> 
>> >  	"github.com/docker/notary/pkg/passphrase"
>> >  	"github.com/docker/notary/version"
>> >  )
>> >  
>> > -const configFileName string = "config"
>> > -const defaultTrustDir string = ".notary/"
>> > -const defaultServerURL = "https://notary-server:4443"
>> > -const idSize = 64
>> > +const (
>> > +	configFileName   string = "config"
>> > +	defaultTrustDir  string = ".notary/"
>> > +	defaultServerURL        = "https://notary.docker.io"
>> This is the default notary server in Docker, but for dev I think we just want to be able to point to our local instances so this should be https://notary-server:4443
>> 
>> —
>> Reply to this email directly or view it on GitHub.
>>